jqassistant/index.adoc

```diff
@@ -26,6 +26,39 @@ LIMIT 1
 ----
 // end::designsmellGod[]
 
+==== Passive Abstraction (Data class)
+// tag::designsmellPassiveAbstraction[]
+[[designsmell:PassiveAbstraction]]
+[source,cypher,role=concept]
+.A class declares fields without any methods operating on it.
+----
+MATCH
+  (field:Field)<-[:DECLARES]-(class:Class)-[:DECLARES]->(method:Method)
+WHERE NOT
+  (method)-[:READS|:WRITES]->(field)
+AND
+  (class)<-[:CONTAINS]–(:Package {name: "passive"})
+RETURN
+  class AS Class, field AS Field, method AS `used by`
+----
+// end::designsmellPassiveAbstraction[]
+
+// tag::designsmellPassiveAbstractionAdvanced[]
+[[designsmell:PassiveAbstractionAdvanced]]
+[source,cypher,role=concept,reportType="plantuml-class-diagram"]
+.Advanced: query result produces class diagram
```

Comment:
Please rename it to "example case" (or the like) to differentiate it from the original idea of "advanced queries" (which are optimized queries). Also just search for the Class "DataHolder", so that we can show one UML diagram for this design smell example.

Reply:
> Please rename it to "example case" (or the like)

done.